test/xref2/github_issue_1001.t/run.t

```diff
@@ -0,0 +1,28 @@
+  $ ocamlc -c -bin-annot test.ml
+  $ odoc compile test.cmt
+  $ odoc link test.odoc
+  $ odoc html-generate -o html test.odocl
+  $ odoc support-files -o html
+
+We should have an 'Optional' argument (as opposed to a 'RawOptional' one)
+
+  $ odoc_print -r f test.odocl
```

Comment:
I think it could be useful to have the problematic case as well. I added it here: https://github.com/jonludlam/odoc/pull/48

Reply:
thanks!